dashboard/app/controllers/api/v1/sections_students_controller.rb

```diff
@@ -16,6 +16,7 @@ def index
   # PATCH /sections/<section_id>/student/<id>/update
   def update
     student = @section.students.find_by_id(params[:id])
+    return render json: {errors: 'Invalid request'}, status: :bad_request if student.nil?
```

Comment:
I think a 404 (not found) would be more appropriate here, since the url contains a student id that is not in the section.

Prefer something like  `return head: :not_found unless student` or `return render_404 unless student`